### ISSUE TITLE
Error when running ping command

### ISSUE THREAD

**Issue by vineet-sinha** [matrix-io/matrix-cli#105]
Lines https://github.com/matrix-io/matrix-cli/blob/master/lib/app.js#L163-L164 need to only run after a conditional check on `err`.

**Comment by hpsaturn** [comment]
Maybe is the same error like this:

`matrix ping`
______
```javascript
Matrix CLI Error: [TypeError: Cannot read property 'message' of null] TypeError: Cannot read property 'message' of null
    at /usr/local/lib/node_modules/matrix-cli/lib/app.js:163:28
    at /usr/local/lib/node_modules/matrix-cli/node_modules/async/lib/async.js:726:13
    at /usr/local/lib/node_modules/matrix-cli/node_modules/async/lib/async.js:52:16
    at /usr/local/lib/node_modules/matrix-cli/node_modules/async/lib/async.js:269:32
    at /usr/local/lib/node_modules/matrix-cli/node_modules/async/lib/async.js:44:16
    at /usr/local/lib/node_modules/matrix-cli/node_modules/async/lib/async.js:723:17
    at /usr/local/lib/node_modules/matrix-cli/node_modules/async/lib/async.js:167:37
    at Socket.<anonymous> (/usr/local/lib/node_modules/matrix-cli/node_modules/matrix-node-sdk/utils/Socket.js:28:13)
    at Socket.Emitter.emit (/usr/local/lib/node_modules/matrix-cli/node_modules/component-emitter/index.js:133:20)
    at Socket.onClose (/usr/local/lib/node_modules/matrix-cli/node_modules/engine.io-client/lib/socket.js:715:10)
```

**Comment by gordinh** [comment]
Yes it's the same. Thanks @hpsaturn and @vineet-sinha

**Comment by chbardel** [comment]
I get the same error! How can I workaround this?

**Comment by gordinh** [comment]
@chbardel for now you can update your CLI and work in `dev` branch.

**Comment by LexiconCode** [comment]
@gordinh 
Could you post the commands to switch **and** update to dev branch?

**Comment by gordinh** [comment]
`git checkout dev` and you can do a `git pull` to update

**Comment by LexiconCode** [comment]
@gordinh 

upgrade matrix-os

I've tried following the documentation [here](https://github.com/matrix-io/matrix-documentation/blob/master/docs/upgrades.md) regarding upgrading. I've checked out the dev and master branch to which it seems to be up-to-date.

Unfortunately MATRIX OS reports the following..

"v0.17.1-22211c5
MATRIX OS can be upgraded. 0.17.0 available! 0.17.1"

Yet even that is inaccurate 1.7.2 seems to be available.
https://github.com/matrix-io/matrix-cli/releases/tag/v1.7.2
